app/jobs/deposit_job.rb

```diff
@@ -32,6 +34,14 @@ def perform_login
     raise login_result.failure unless login_result.success?
   end
 
+  def disallow_writes(work_version)
+    return true if test_mode?
+    return true if integration_test_mode? && work_version.globus_endpoint == integration_endpoint
+
+    # user_id nil because unneeded for permission update operations
+    GlobusClient.disallow_writes(path: work_version.globus_endpoint, user_id: nil)
+  end
```

Comment:
presume this method is already in the client gem used here, right?

Reply:
Yes, method was already in the gem, just unused.